### ISSUE TITLE
Export menu items not announced by JAWS

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#9714]
#### Expected behaviour
Screen readers should read out export menu items as they are navigated.

#### Actual behaviour
According to user, JAWS is not reading the menu items. Suggestion from user: changing the role attribute value on export menu element to application list and changing role  attribute value of single item to listitem.

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/accessible-pie/

**Comment by oysteinmoseng** [comment]
This seems to be an issue now with JAWS 2018, where it is not able to click the export button. JAWS 17 is working fine. We should also test with JAWS 2019 and see if this was simply a regression from JAWS' side.

All versions are able to navigate the export menu by tabbing to it.

**Comment by oysteinmoseng** [comment]
The issue with clicking the button will be fixed in v7.1.

JAWS 19 still has problems navigating the menu after open it (unless you tab to it). This seems to be a JAWS bug, as the cursor simply gets stuck, not being able to move anywhere. Other screen readers seem to be able to navigate the menu without issue. Changing the roles of the menu to use `list` and `listitem` did not seem to help.
